sdk/resourcemanager/azure-resourcemanager-appplatform/src/main/java/com/azure/resourcemanager/appplatform/implementation/SpringAppImpl.java

```diff
@@ -160,10 +160,21 @@ public boolean hasConfigurationServiceBinding() {
             return false;
         }
         return addonConfigs.get(Constants.APPLICATION_CONFIGURATION_SERVICE_KEY) != null
-            && Objects.equals(
-            addonConfigs.get(Constants.APPLICATION_CONFIGURATION_SERVICE_KEY).get(Constants.BINDING_RESOURCE_ID),
-            configurationService.id()
-        );
+            && configurationService.id().equalsIgnoreCase((String) addonConfigs.get(Constants.APPLICATION_CONFIGURATION_SERVICE_KEY).get(Constants.BINDING_RESOURCE_ID));
```

Comment:
why we use equalsIgnoreCase?

Reply:
I just learnt that resource names are case-insensitive according to the doc: https://docs.microsoft.com/en-us/azure/azure-resource-manager/management/resource-name-rules, so I changed it just in case.